src/components/CollectionsAndPalettesFunnel/CollectionsAndPalettesFunnel.tsx

```diff
@@ -0,0 +1,55 @@
+import {
+  COLLECTIONS_URL,
+  PALETTES_URL,
+  COLLECTIONS_AND_PALETTES_URL,
+} from "constants/urls";
+import * as React from "react";
+import {
+  Link,
+  Redirect,
+  Route,
+  RouteComponentProps,
+  Switch,
+} from "react-router-dom";
+import styles from "./CollectionsAndPalettesFunnel.module.scss";
+import { AppContext, AppContextState } from "../../context/AppContext";
+import Breadcrumbs from "../common/Breadcrumbs/Breadcrumbs";
+import { BREADCRUMBS_STEPS } from "../../constants/general";
+import { isOnMobile } from "../../utils/responsive";
+import LogoMobile from "../common/LogoMobile/LogoMobile";
+import Collections from "./Collections/Collections";
+import Palettes from "./Palettes/Palettes";
+
+type Props = RouteComponentProps;
+
+export default class CollectionsAndPalettesFunnel extends React.Component<Props> {
```

Comment:
Not sure this is a funnel, so I think we can just name the component + folders `CollectionsAndPalettes`